readme.md

```diff
@@ -190,6 +190,56 @@ if (additionalPosts.length) {
 }
 ```
 
+
+#### 📍 no-implicit-coercion
+Encourages stopping mixing different types of variables for the sake of cleaner and more readable code.
+
+##### ❌ Example of incorrect code for this rule:
+
+```
+// Boolean
+const b = !!foo;
+const b = ~foo.indexOf(".");
+
+// Number
+const n = +foo;
+const n = 1 * foo;
+
+// Strings
+const s = "" + foo;
+const s = `` + foo;
+foo += "";
+foo += ``;
+
+```
+
+##### ✅ Example of correct code for this rule:
+
+```
+// Boolean
+const b = Boolean(foo);
+const b = foo.indexOf(".") !== -1;
+
+// Number
+const n = Number(foo);
+const n = parseFloat(foo);
+const n = parseInt(foo, 10);
+
+// Strings
+const s = String(foo);
+foo = String(foo);
+```
+
+or, if the value can be changed
+
+```
+let count = posts.length;
```

Comment:
What would this example replace? What would the "incorrect" way be?

Reply:
Oops, this was actually accidentally copied from the rule above.